.github/workflows/ci.yml

```
@@ -54,11 +54,11 @@ jobs:
         with:
           python-version: ${{ matrix.python_version }}
 
-      - name: Install molecule and openshift dependencies
-        run: pip install ansible~=2.9.15 molecule yamllint openshift flake8
+      - name: Install molecule and kubernetes dependencies
+        run: pip install ansible~=2.9.15 molecule yamllint kubernetes flake8
```

Do we need to update the ansible version for this job as well?
```suggestion
        run: pip install ansible~=2.15.0 molecule yamllint kubernetes flake8
```